src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -323,7 +323,7 @@
   "block.structurize.side_framed_horizontal_jungle_brick_timber_frame": "Side Framed Horizontal Jungle Brick Timber Frame",
   "block.structurize.side_framed_cactus_oak_timber_frame": "Side Framed Cactus Oak Timber Frame",
   "block.structurize.gated_cactus_birch_timber_frame": "Gate Framed Cactus Birch Timber Frame",
-  "structurize.config.maxcachedchanges.comment": "Max amount of changes cached to be able to undo",
+  "structurize.config.maxcachedchanges.comment": "Max amount of changes to be able to undo cached.",
```

Comment:
Probably might want to rephrase that entirely. To something like "Max amount of undos. Higher number require more memory" (feel free to make up your own version :) )